pylint/checkers/python3.py

```diff
@@ -1039,7 +1041,7 @@ def visit_name(self, node):
         self.add_message(message, node=node)
 
     @utils.check_messages("print-statement")
-    def visit_print(self, node):
+    def visit_print(self, node) -> None:
         self.add_message("print-statement", node=node, always_warn=True)
```

Comment:
Can be removed completely. Parsing legacy code is no longer supported by astroid.
Maybe `visit_print` and `visit_exec` statements should be removed in a separate PR prior to this one?

Reply:
See #4942